components/gitpod-protocol/src/billing-mode.ts

```diff
@@ -41,8 +41,8 @@ export namespace BillingMode {
             return false;
         }
 
-        // if has any Stripe subscription, either directly or per team
-        return billingMode.mode === "usage-based";
+        // if has any Stripe subscription, either directly or per team, OR we're running on a license
+        return billingMode.mode === "usage-based" || billingMode.mode === "none";
```

Comment:
@geropl PTAL as you've got the most context on the billing modes

Reply:
I authored this change :heavy_check_mark: